### PR TITLE
i37 - Retrieve status endpoint from Newslabs STT service

### LINKED ISSUE #37: Integrate with PSTT: Retrieve status of transcript by accessing an endpoint
**Is your feature request related to a problem? Please describe.**
- [x] Retrieve status by accessing an endpoint provided by Ashley.

POST to endpoint, request body like so: `{"objectKey": "dpe/users/gLoKESQNkVavUfSiqqoSEEEa6Lo2/audio/1ipL5m7V7Jal7h3x3Odu.wav"}` with api key in the `X-API-Key` header!

This may require some tweaking of AWS Roles / Users, depending on what Ashley decides to use. We do NOT want to reuse the existing user that uploads to S3 in https://github.com/bbc/digital-paper-edit-firebase/issues/28

Out of scope:
-  Store status in Transcripts' documents (see https://github.com/bbc/digital-paper-edit-firebase/issues/57)
- Store transcript in Transcripts' documents (see https://github.com/bbc/digital-paper-edit-firebase/issues/30)

**Describe the solution you'd like**
We can check the status and retrieve transcription.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by emettely**
**Is your Pull Request request related to another [issue](https://github.com/bbc/react-transcript-editor/issues) in this repository ?**      
Fixes #37 

**Describe what the PR does**    
- [x] Add new executable script to start local emulators for functions
- [x] Updated files related to the runtime config: example file and audioStripper to fit what we have now
- [x] general cleanup: `functions/audioStripper/index.js`
- [x] implemented getting Status from STT - see `functions/sttChecker/index.js`
- [x] added helper functions for `sttChecker`

**State whether the PR is ready for review or whether it needs extra work**    
Ready

**Additional context**

### CODE REVIEW

File: functions/utils/index.js [hunk 1]
```diff
@@ -13,4 +13,71 @@ const getUrl = async srcFile => {
   }
 };
 
+const secondsToDhms = (seconds) => {
+  seconds = Number(seconds);
+  var d = Math.floor(seconds / (3600 * 24));
```
Comment (bennuttall) [inline functions/utils/index.js:18]: Any reason these are `var` not `let`? Maybe just copy/paste?

**Comment by bennuttall** [comment]
Looks good at a first glance but I'll check it out functionally in the morning.

File: functions/utils/index.js (hunk 1 above)
Comment (emettely) [reply functions/utils/index.js:18]: 😂 yup

File: functions/utils/index.js [hunk 2]
```diff
@@ -13,4 +13,71 @@ const getUrl = async srcFile => {
...
+  var h = Math.floor((seconds % (3600 * 24)) / 3600);
+  var m = Math.floor((seconds % 3600) / 60);
+  var s = Math.floor(seconds % 60);
+
+  var dDisplay = d > 0 ? d + (d == 1 ? " day, " : " days, ") : "";
+  var hDisplay = h > 0 ? h + (h == 1 ? " hour, " : " hours, ") : "";
+  var mDisplay = m > 0 ? m + (m == 1 ? " minute, " : " minutes, ") : "";
+  var sDisplay = s > 0 ? s + (s == 1 ? " second" : " seconds") : "";
```
Comment (emettely) [inline functions/utils/index.js:26]:
```suggestion
  const d = Math.floor(seconds / (3600 * 24));
  const h = Math.floor((seconds % (3600 * 24)) / 3600);
  const m = Math.floor((seconds % 3600) / 60);
  const s = Math.floor(seconds % 60);

  const dDisplay = d > 0 ? d + (d == 1 ? " day, " : " days, ") : "";
  const hDisplay = h > 0 ? h + (h == 1 ? " hour, " : " hours, ") : "";
  const mDisplay = m > 0 ? m + (m == 1 ? " minute, " : " minutes, ") : "";
  const sDisplay = s > 0 ? s + (s == 1 ? " second" : " seconds") : "";
```

**Comment by emettely** [comment]
Review first out of #39, #59, #61

**Pull request merged**